tests/test_null.py

```diff
@@ -3,23 +3,70 @@
 import pytest
 from preprocessy.handlenullvalues import NullValuesHandler
 
+dataframe = pd.read_csv("datasets/encoding/test2.csv")
+
 
 class TestHandlingNullValues:
     def test_null_dataframe(self):
         with pytest.raises(ValueError):
             handler = NullValuesHandler()
             handler.execute()
 
-    def test_incorrect_input_type(self):
-        self.dataframe = pd.read_csv("datasets/encoding/testnew.csv")
-        with pytest.raises(TypeError):
-            handler = NullValuesHandler(df=self.dataframe, drop="nice")
+    def test_none_args(self):
+        with pytest.raises(Exception):
```

Comment:
I think we can catch `ArgumentsError` as well. You'll have to import `ArgumentsError` class. The same applies for other tests as well. I'll mark them for your reference.

Please refer to this [stackoverflow post](https://stackoverflow.com/questions/15012539/using-pytest-raises-to-catch-expected-custom-error)

Reply:
Sure, will make the changes